app/services/open_investment_time_calculator.rb

```diff
@@ -0,0 +1,39 @@
+class OpenInvestmentTimeCalculator
+  def initialize
+    @updated_times_of_employees = nil
+
+    @employee = { redmine_id: nil, entries: nil, investment_time: nil, employee_from_db: nil }
```

Comment:
The concept of having this class to find the *Employee* on it's own leads to more coupling than needed. The only thing of the *Employee* we need in this class here if `open_investment_time`. You could do something like that:

    calculator = OpenInvestmentTimeCalculator.new(investment_time_balance)
    calculator.new_balance(time_entries)

And then assign this new balance to the user. The initialization could even have a default parameter being `0`.